scripts/webpack/webpack.config.ts

```diff
@@ -103,7 +103,10 @@ const webpackConfig: webpack.Configuration = {
       contextRegExp: /moment$/,
     }),
     __DEV__ &&
```

Comment:
just checking: you mention in the comment "in CI" but here checks for dev environment.